p8e-api/src/main/kotlin/io/provenance/engine/domain/Transaction.kt

```diff
@@ -0,0 +1,111 @@
+package io.provenance.engine.domain
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import feign.Param
+import feign.RequestLine
+import io.provenance.engine.stream.domain.*
+import org.springframework.stereotype.Component
+
+data class BlockchainInfo(
```

Comment:
Note: I explored getting rid of these data classes I created and instead parsing the json-rpc call responses back to the new cosmos protobuf responses in order to more seamlessly prepare for when we are developing against the public chain. However, there was really no clean mapping of messages to do this conversion, there were slight differences all over the place (either properties nested at a different level or slightly differently named properties)... so it seemed like maybe more trouble than it is worth for a bit of short term gain

Reply:
Sounds good.